tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -0,0 +1,688 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
How is this tested? Is it possible to reuse the tests here? tensorflow/python/ops/nn_fused_batchnorm_test.py